app/javascript/components/data-tables/optimization/helper.js

```diff
@@ -5,12 +5,12 @@ const convertHeader = (headerItems) => headerItems.map((item) => ({ key: item[0]
 
 /** Attributes needed for a button. */
 const buttonData = (item) => ({
-  alt: 'Queue Report',
+  alt: `${__('Queue Report')}`,
```

Comment:
Hey @GilbertCherrie , can't we just use like this - `__('Queue Report')`

Reply:
@jeffibm Should be fixed now if you can review again